### DISCUSSION TITLE
[Feature] Add Proxy Support

### DISCUSSION THREAD

**Discussion by ghbjklhv** [hyperspacedev/hyperspace#66] [Ideas]
**Feature**: Add proxy support to settings.
**Why?**: Many reasons, in my case this would be useful for routing Mastadon through [TOR](https://www.wikipedia.org/wiki/Tor_(anonymity_network)) but it could also be used for [I2p](https://www.wikipedia.org/wiki/I2P) or a regular clear net [proxy](https://www.wikipedia.org/wiki/Proxy_server) or anything else that supports [proxy](https://www.wikipedia.org/wiki/Proxy_server).
It would also allow users to connect to a [.onion](https://www.wikipedia.org/wiki/.onion) and [I2p](https://www.wikipedia.org/wiki/I2P#Terminology) Mastadon instances.

__________________________

Want to support this issue? Post a bounty on it! We accept bounties via [BountySource](https://www.wikipedia.org/wiki/Bountysource).

[a comment by an automated account is not shown]

**Comment by alicerunsonfedora** [comment]
We'll definitely take a look, though proxy support probably won't make it to the web version. It may be possible with the desktop apps via some Electron APIs, though I can't guarantee it.
